client/src/apps/match/MatchApp.tsx

```diff
@@ -1,9 +1,40 @@
+{/*This is used for testing. it will be deleted later */}
+// import {matchApp} from "server/src/Schema";
+import { postJson } from "../../util";
+import {MatchData} from  "server/requests"
+
 function MatchApp() {
+    
+
+    {/*This is used for testing. it will be deleted later */}    
+    const test = async () => {
+        const data: MatchData = ({
+            metadata:{
+                scouterName: "bhhjb",
+                robotTeam: 23,
+                robotPosition: 'red_2'
+            },
+            nonAmpedSpeakerNotes: 121,
+            ampedSpeakerNotes: 39,
+            ampNotes: 321,
+            trapNotes: 45453,
+            highNotes: 5443,
+            climb: 'success',
+            parked: true,
+            disabledSeconds: 0
+        })
+        await postJson('http://localhost:8080/data/match', data);
```

Comment:
I think you can change this back to just `'/data/match'`, it was probably not working because you went to localhost:5173 instead of localhost:8080
```suggestion
        await postJson('/data/match', data);
```